plugins/scoreboard/derma/cl_scoreboard.lua

```diff
@@ -94,6 +94,12 @@ local PANEL = {}
 				elseif (!visible and amount > 0) then
 					v:SetVisible(true)
 				end
+				
+				local faction = nut.faction.indices[k]
+				
+				if (amount != 0) then
+					v:SetVisible(hook.Run("HideFactionInScoreboard", faction) or LocalPlayer():IsAdmin())
```

Comment:
```suggestion
					v:SetVisible(hook.Run("HideFactionInScoreboard", faction) ~= false or LocalPlayer():IsAdmin())
```

Reply:
why

Reply:
https://github.com/NutScript/NutScript/pull/42#discussion_r605895303
that's why

Reply:
Should probably change HideFactionInScoreboard to ShowFactionInScoreboard, since you return false to hide it.